src/encoded/static/components/glossary/glossary.json

```diff
@@ -1,10 +1,512 @@
 [
     {
-		"term": "Functional Genomics data",
+		"term": "functional genomics data",
 		"definition": "Data generated by assays investigating processes such as transcription, translation and epigenetic regulation on a genome-wide scale. Examples of assays generating functional genomics data: RNA-seq, ChIP-seq, DNase-seq, ATAC-seq, WGBS, HiC, and ChIA-PET."
 	},
 	{
-		"term": "Functional Characterization data",
+		"term": "functional characterization data",
 		"definition": "Data generated by assays (i.e STARR-seq, MPRA, and CRISPR screen) investigating the relationship between DNA sequences and their regulatory activities."
```

Comment:
@iyoungworth can  you please align these two old entries with the rest of the entries in the file? It looks like the indentation of these entries is different.

Reply:
Sure thing- it looks like my text editor switched between tabs and spaces. They should all be consistent now.